robot/power.py

```diff
@@ -7,6 +8,18 @@
 LOGGER = logging.getLogger(__name__)
 
 
+# Keep this in sync with `robotd`
+class PowerOutput(enum.Enum):
```

Comment:
We need to expose this as part of the API -- we'll need to import it into `__init__.py` and re-export it from there.

Reply:
a6f08ea